backend/controllers/taskController.js

```diff
@@ -0,0 +1,48 @@
+categories = {
+    'Work': 5,
+    'Personal': 3,
+    'Health': 7,
+    'Education': 6,
+    'Entertainment': 2,
+    'Family': 8,
+    'Errands': 4,
+    'Finance': 6,
+    'Social': 3,
+    'Fitness': 9,
+    'Hobbies': 4,
+    'Volunteering': 7,
+    'Travel': 6,
+    'Skills Development': 5,
+    'Reading': 3,
+    'Projects': 8,
+    'Meetings': 5,
+    'Appointments': 4,
+    'Chores': 6,
+    'Research': 7,
+    'Creativity': 5,
+    'Cooking': 4,
+    'Learning': 6,
+    'Home improvement': 7,
+    'Mental well-being': 8,
+    'Event planning': 5,
+    'Outdoor tasks': 6,
+    'Self-reflection': 4,
+    'Music practice': 7,
+    'Social media management': 5,
+    'Writing': 6,
+    'Relaxation': 4,
+    'Watching TV': 3,
+    'Socializing': 5,
+    'Gardening': 7,
+    'Shopping': 6
+}
```

Comment:
Do we really want to add so many categories? They are quite fragmented, and in my opinion, some of them could be combined for simplification, for example:
Education, Learning, Skills Development -> Education
Family, Social, Meetings, Appointments, Socializing -> Social

I assume that the breakdown into so many categories was intended to make the prioritization feature more interesting and precise. However, let's keep in mind that the user will have to choose categories independently when creating a task. I think a list of 35 categories might be a bit overwhelming for the user :) I would suggest limiting it to around 10 categories.